posthog/api/prompt.py

```diff
@@ -1,93 +1,231 @@
-from typing import Any, Dict, List, Optional
```

Comment:
For reviewers, this is the main file to check, in particular:
- New serializers
- `my_prompts` endpoint has been refactored to use the new models, no major logic changes there
- New `webhook endpoint`